configs/defs/generic_linux.textproto

```diff
@@ -9558,3 +9558,931 @@ benchmark_configs: {
+    description:
+      "The max_log_file_action setting determines how to handle the audit log file reaching the "
+      "max file size. A value of keep_logs will rotate the logs but never delete old logs."
+    rationale:
+      "In high security contexts, the benefits of maintaining a long audit history exceed the cost of "
+      "storing the audit history."
+    remediation:
+      "Set the following parameter in /etc/audit/auditd.conf:\n"
+      "```\n"
+      "max_log_file_action = keep_logs\n"
+      "```"
+    cis_benchmark: {
+      profile_level: 2
+      severity: LOW
+    }
+    scan_instructions:
+      "generic:{"
+      "      check_alternatives:{"
+      "        file_checks:{"
+      "          files_to_check:{single_file:{path:\"/etc/audit/auditd.conf\"}}"
+      "              content_entry:{"
+      "                match_type: ALL_MATCH_ANY_ORDER"
+      "                match_criteria: {"
+      "                  filter_regex: \"\\\\s*max_log_file_action(\\\\s|=).*\""
+      "                  expected_regex: \"max_log_file_action\\\\s*=\\\\s*keep_logs\""
+      "                }"
+      "              }"
+      "        }"
+      "    }"
+      "}"
+  }
+}
+
+
+
+benchmark_configs: {
+  id: "audit-log-storage-size-configured"
+  compliance_note: {
+    title: "Ensure audit log storage size is configured"
+    description:
+      "Configure the maximum size of the audit log file. "
+      "Once the log reaches the maximum size, it will be rotated and a new log file will be started."
+    rationale:
+      "It is important that an appropriate size is determined for log files "
+      "so that they do not impact the system and audit data is not lost."
+    remediation:
+      "Set the following parameter in /etc/audit/auditd.conf in accordance with site policy: "
+      "```max_log_file = <MB>``` \n"
+      "Notes: The max_log_file parameter is measured in megabytes. \n"
+      "Other methods of log rotation may be appropriate based on site policy. "
+      "One example is time-based rotation strategies which don't have native support in auditd configurations. "
+      "Manual audit of custom configurations should be evaluated for effectiveness and completeness."
+    cis_benchmark: {
+      profile_level: 2
+      severity: LOW
+    }
+    scan_instructions:
+      "generic:{"
+      " check_alternatives:{"
+      "   file_checks:{"
+      "     files_to_check:{single_file:{path:\"/etc/audit/auditd.conf\"}}"
+      "         content_entry:{"
+      "           match_type: ALL_MATCH_ANY_ORDER"
+      "           match_criteria: {"
+      "             filter_regex: \"\\\\s*max_log_file\\\\s*=.*\""
+      "             expected_regex: \"max_log_file\\\\s*=\\\\s*(\\\\d+)\""
+      "           }"
+      "         }"
+      "   }"
+      " }"
+      " check_alternatives:{"
+      "  file_checks:{"
+      "    files_to_check:{single_file:{path:\"/etc/audit/auditd.conf\"}}"
```

Comment:
Why do we have a check_alternative that the file doesn't exist? If the config file isn't there I assume we wouldn't have a maximum size configured